code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/OptimizeExtension.java

```diff
@@ -61,11 +62,13 @@ public boolean doWork(final Event event) {
 
     // Map containing the update event IDs (and corresponding requested scopes) for Edge events that
     // haven't yet received an Edge completion response.
-    private Map<String, List<DecisionScope>> updateRequestEventIdsInProgress = new HashMap<>();
+    private static final Map<String, List<DecisionScope>> updateRequestEventIdsInProgress =
```

Comment:
nit: can you add a comment here on why we need this map to be of type `ConcurrentHashMap`? This would help anyone reading the code and ensure we don't accidentally change it back to non-thread safe implementation

Reply:
done @spoorthipujariadobe